.travis.yml

```diff
@@ -35,9 +35,14 @@ matrix:
     - os: linux
       language: c
       compiler: clang
-      # gcc also works, but to keep the # of concurrent builds down, we use one C
-      # compiler here and the other to run the coverage build. Clang is preferred
-      # in this instance for its better error messages.
+      env: TESTING=cpython
+      addons:
+        apt:
+          packages:
+            - xvfb
+    - os: linux
+      language: c
+      compiler: gcc
```

Comment:
Why you added this?

Reply:
As I mentioned:
> The optional test coverage builds shouldn't be relied upon to certify
that gcc builds don't fail.

It's possible I'm missing something, but adding additional builds is cheap; there's not really a reason _not_ to have them.

Reply:
> The optional test coverage builds shouldn't be relied upon to certify
that gcc builds don't fail.

Note that not including gcc in travis job doesn't mean we don't test gcc.
For example, we use gcc in Ubuntu on Azure Pipelines.  (see [this build](https://dev.azure.com/Python/cpython/_build/results?buildId=41875))
And many buildbot works after merge.  gcc is tested well.


> It's possible I'm missing something, but adding additional builds is cheap; there's not really a reason _not_ to have them.

Additional build is not cheap always.   We experienced long queue waiting many times.

At least, it is not relating to bpo-36684.  Please remove it.

Reply:
Done.